packages/get-platform/src/__tests__/getSSLVersion.test.ts

```diff
@@ -6,11 +6,26 @@ const describeIf = (condition: boolean) => (condition ? describe : describe.skip
 const ctx = jestContext.new().assemble()
 
 describeIf(process.platform === 'linux')('computeLibSSLSpecificPaths', () => {
-  // eslint-disable-next-line jest/no-identical-title
-  it('should not return an error', () => {
+  const OLD_ENV = process.env
+
+  afterAll(() => {
+    process.env = OLD_ENV // Restore old environment
+  })
+
+  it('should return system specific search paths', () => {
+    process.env.LD_LIBRARY_PATH = ''
```

Comment:
maybe

```suggestion
    delete process.env.LD_LIBRARY_PATH
```

Or perhaps even two separate tests for missing and empty `LD_LIBRARY_PATH`?